src/core/checkedint.d

```diff
@@ -492,8 +494,8 @@ unittest
 pragma(inline, true)
 ulong mulu(ulong x, ulong y, ref bool overflow)
 {
-    ulong r = x * y;
-    if (x && (r / x) != y)
+    const r = x * y;
+    if (x && r / x != y)
```

Comment:
If it improves clarity, then I'm in favour.  But there is no excuse to remove them.